addins/cftester/cfActiveTester.xml

```diff
@@ -0,0 +1,158 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<!--Created:ce44715c-8c4e-446b-879c-ea9ebe0f09c8-->
+<OfficeApp
+          xmlns="http://schemas.microsoft.com/office/appforoffice/1.1"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xmlns:bt="http://schemas.microsoft.com/office/officeappbasictypes/1.0"
+          xmlns:ov="http://schemas.microsoft.com/office/taskpaneappversionoverrides"
+          xsi:type="TaskPaneApp">
+
+	<!-- Begin Basic Settings: Add-in metadata, used for all versions of Office unless override provided. -->
+
+	<!-- IMPORTANT! Id must be unique for your add-in, if you reuse this manifest ensure that you change this id to a new GUID. -->
+	<Id>47eadcf3-a3df-40b7-8eaf-8709b1154c19</Id>
+
+	<!--Version. Updates from the store only get triggered if there is a version change. -->
+	<Version>1.0.0.4</Version>
+	<ProviderName>Microsoft Excel Custom Functions</ProviderName>
+	<DefaultLocale>en-US</DefaultLocale>
+	<!-- The display name of your add-in. Used on the store and various places of the Office UI such as the add-ins dialog. -->
+	<DisplayName DefaultValue="cfActiveTester" />
```

Comment:
Is literally 'cfActiveTester' what will appear on the task pane? If so, please spell it out properly.